### PR TITLE
PMM-8514 refactoring

### PR DESCRIPTION
**Pull request by BupycHuk**
https://github.com/percona/pmm/pull/779
https://github.com/percona/pmm-managed/pull/832
https://github.com/percona-platform/dbaas-controller/pull/224
https://github.com/percona-platform/dbaas-api/pull/167
https://github.com/Percona-Lab/pmm-submodules/pull/2046

### CODE REVIEW

**Review by jprukner — APPROVED** [review]

**Review by tiagomotasantos — APPROVED** [review]

**Review by tiagomotasantos — COMMENTED** [review]
Created https://jira.percona.com/browse/PMM-8956 subtask two handle the change in the frontend.

The only change is that the list request now returns PXC and PSMDB clusters right?

**Comment by BupycHuk** [comment]
@tiagomotasantos,I also renamed xtradb to pxc everywhere. except the places where we use it as `Percona XtraDB Cluster`.

**Review by JiriCtvrtka — APPROVED** [review]

**Review by JiriCtvrtka — DISMISSED** [review]

**Review by jprukner — DISMISSED** [review]

File: api/managementpb/dbaas/db_clusters.proto [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  rpc ListDBClusters(ListDBClustersRequest) returns (ListDBClustersResponse) {
+    option (google.api.http) = {
+      post: "/v1/management/DBaaS/DBClusters/List"
+      body: "*"
+    };
+  }
+  // RestartDBCluster restarts DB cluster.
+  rpc RestartDBCluster(RestartDBClusterRequest) returns (RestartDBClusterResponse) {
+    option (google.api.http) = {
+      post: "/v1/management/DBaaS/PXCCluster/Restart"
```
Comment (tiagomotasantos) [inline api/managementpb/dbaas/db_clusters.proto:144]:
```suggestion
      post: "/v1/management/DBaaS/DBClusters/Restart"
```

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: api/managementpb/dbaas/dbaas.proto [hunk 2]
```diff
@@ -35,6 +35,16 @@ message Resources {
   uint64 disk_size = 3;
 }
 
+// DBClusterType represents database cluster type.
+enum DBClusterType {
+  // DB_CLUSTER_TYPE_INVALID represents unknown cluster type.
+  DB_CLUSTER_TYPE_INVALID = 0;
+  // DB_CLUSTER_TYPE_PXC represents pxc cluster type.
+  DB_CLUSTER_TYPE_PXC = 1;
+  // DB_CLUSTER_TYPE_PXC represents psmdb cluster type.
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply api/managementpb/dbaas/dbaas.proto:44]: 👍🏼

**Review by palash25 — APPROVED** [review]

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**